rock_gazebo.orogen

```diff
@@ -49,10 +49,17 @@ task_context "ModelTask" do
     # The covariance matrix to be used as velocity covariance. Default is unset (NaN)
     property 'cov_velocity', '/base/Matrix3d'
 
+    # The acceleration of the model w.r.t. the world frame
+    output_port 'ang_accel_samples', '/base/samples/RigidBodyAcceleration'
+    # The covariance matrix to be used as acceleration covariance. Default is unset (NaN)
+    property 'cov_acceleration', '/base/Matrix3d'
+
     property "exported_links", "std/vector<rock_gazebo/LinkExport>"
 
     # The model's pose if one wants to "warp"
     input_port 'model_pose', "/base/samples/RigidBodyState"
+    # Apply wrench to the model
+    input_port 'wrenches', "/base/samples/Wrenches"
```

Comment:
> it would be good to have one RigidBodyAcceleration with linear and angular.

Let's try that. Change the RBA type and create a pull request on base/types. Let's see how people react.

Do not change the name of the existing fields, it ensures a less painful migration.